static/app/components/feedback/widget/useFeedbackWidget.tsx

```diff
@@ -7,7 +7,7 @@ import ConfigStore from 'sentry/stores/configStore';
 import {useLegacyStore} from 'sentry/stores/useLegacyStore';
 
 interface Props {
-  buttonRef?: RefObject<HTMLButtonElement>;
+  buttonRef?: RefObject<HTMLButtonElement> | RefObject<HTMLAnchorElement>;
```

Comment:
```suggestion
  buttonRef?: RefObject<HTMLButtonElement | HTMLAnchorElement>;
```

can't it be written like this

Reply:
Oh really? What's the difference in this scenario?

Reply:
Yeah, correct. Hopefully this [explains](https://www.typescriptlang.org/play?#code/JYWwDg9gTgLgBDAnmApnA3gJRQMwPIBGAVigMYwC+cOUEIcARFCgIbkMBQHOArgHblgEPnADuwACYoAFMxwAuONnzEyMADwAJACoBZADIAhHjBjCAogBsUIFHxgAfHQYCCAgBbQrNuzAB8AJToFNz8gsJwfCxQtKKyuIrKhCTkWnpGJmZ83rb2fnAOSrjJammuHl7Wuf5BIRykwgDO8OJSAPpyicWqqc4ZphZVvk7pbqSeUDm++QC8GHCkPDG+inw8lpYhDXzNkdGxHQlFKikafcYD2UN5cHPoC0vM9qvrm1ytMh+HOAEcUTEQOJfOS-f6xaRgwHfX5cIA) it

Reply:
Yeah I see what you're saying. Subtle. 👍